yarn.lock

```diff
@@ -227,6 +227,11 @@
   dependencies:
     "@babel/types" "^7.8.3"
 
+"@babel/helper-validator-identifier@^7.10.4", "@babel/helper-validator-identifier@^7.9.0":
+  version "7.10.4"
```

Comment:
Ahh I have no idea why we're getting 7.10 here. Yarn deduping is the worst.

Reply:
No idea! As long as tests are green and bug is fixed, that's legit to me.